### PR TITLE
[DownloadBuildArtifactsV0] Fix security issues

### PR DESCRIPTION
**Pull request by tatyana-kostromskaya**
**Task name**: DownloadBuildArtifactsV0

**Description**: Update version of artifact-engine to latest one which fixed 7 critical security errors

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) #15778 

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/DownloadBuildArtifactsV0/task.loc.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 0,
-    "Minor": 198,
+    "Minor": 199,
```
Comment (anatolybolshakov) [inline Tasks/DownloadBuildArtifactsV0/task.loc.json:12]: Please bump to 200

**Comment by anatolybolshakov** [comment]
@tatyana-kostromskaya could you please describe test cases checked?

**Review by anatolybolshakov — APPROVED** [review]
LGTM in general, thanks!

**Comment by tatyana-kostromskaya** [comment]
> @tatyana-kostromskaya could you please describe test cases checked?

I tested this task on full canary scenario: on different platforms download artifacts with different options

**Review by golubev-ivan — APPROVED** [review]

**Pull request merged**